vault-admin-client/src/test/java/com/scality/osis/vaultadmin/impl/BaseTest.java

```diff
@@ -75,6 +81,73 @@ protected void initMocks() {
       }
     });
 
+    //initialize mock list accounts response
+    when(accountServicesClient.listAccounts(any(ListAccountsRequestDTO.class)))
+            .thenAnswer(new Answer<Response<ListAccountsResponseDTO>>() {
+              @Override
+              public Response<ListAccountsResponseDTO> answer(InvocationOnMock invocation) {
+                ListAccountsRequestDTO request = invocation.getArgument(0);
+                String marker = request.getMarker();
+                int maxItems = request.getMaxItems();
+                String filterKey = request.getFilterKey();
+                String filterKeyStartsWith = request.getFilterKeyStartsWith();
+
+                Map<String, String> customAttributes1  = null ;
+                boolean withcdTenantId = false;
+
+                if(StringUtils.isNotBlank(filterKey)){
+                  maxItems = 1;
+                  customAttributes1 = new HashMap<>();
+                  customAttributes1.put(filterKey,"");
+                }
+
+                if(StringUtils.isNotBlank(filterKeyStartsWith)){
+                  withcdTenantId = true;
+                }
+
+                if(maxItems <= 0){
+                  maxItems = 5; // Test Accounts count DEFAULT
+                }
+
+                int i = 0;
+                int markerVal = 0;
+                if(StringUtils.isNotBlank(marker)){
+                  markerVal = Integer.parseInt(marker.substring(marker.length()-1));
+                  // extracting markerVal index at last character
+                }
+
+                List<AccountData> accounts = new ArrayList<>();
+                // Generate Accounts with ids (markerVal + i) to maxItems count
+                for(; i < maxItems; i++){
+                  AccountData data = new AccountData();
+                  data.setEmailAddress(DEFAULT_TEST_EMAIL_ADDR);
+                  data.setName(DEFAULT_TEST_ACCOUNT_NAME);
+                  data.setArn(DEFAULT_TEST_ARN_STR + DEFAULT_TEST_ACCOUNT_NAME +"/\"");
+                  data.setCreateDate(new Date());
+                  data.setId(DEFAULT_TEST_ACCOUNT_ID + (i + markerVal)); //setting ID with index
+                  data.setCanonicalId(DEFAULT_TEST_CANONICAL_ID);
```

Comment:
You might get another field if quota is set.
It only shows up in listAccounts if quota is set other than zero.
Just like `customAttributes`

Reply:
example:
```js
❯ ./node_modules/vaultclient/bin/vaultclient --config ../admin.conf create-account --name=an2usdr2ag2 --email ans2durag2@test.com  --quota 100 --host 127.0.0.1 --port 8600
{
    "account": {
        "arn": "arn:aws:iam::256476687020:/an2usdr2ag2/",
        "canonicalId": "24e9e24a454ee9a0112fb296fe1e4362eaf79d609961de76ccac37f65a7d9b5c",
        "id": "256476687020",
        "emailAddress": "ans2durag2@test.com",
        "name": "an2usdr2ag2",
        "createDate": "2021-03-26T11:02:52Z",
        "aliasList": [],
        "oidcpList": [],
        "quotaMax": 100,
        "customAttributes": {}
    }
}
❯ ./node_modules/vaultclient/bin/vaultclient --config ../admin.conf list-accounts   --host 127.0.0.1 --port 8600
{
    "isTruncated": false,
    "accounts": [
        {
            "arn": "arn:aws:iam::256476687020:/an2usdr2ag2/",
            "canonicalId": "24e9e24a454ee9a0112fb296fe1e4362eaf79d609961de76ccac37f65a7d9b5c",
            "id": "256476687020",
            "emailAddress": "ans2durag2@test.com",
            "name": "an2usdr2ag2",
            "createDate": "2021-03-26T11:02:52Z",
            "quota": 100
        }
    ]
}
```

Reply:
This is a fix on `vaultclient-java`. Created a PR for this
https://github.com/scality/vaultclient-java/pull/16